zstacklib/zstacklib/utils/iproute.py

```
@@ -530,18 +584,20 @@ def set_link_attribute(ifname_or_index, namespace=None, **attributes):
         if attributes:
             if attributes.has_key('master'):
                 attributes['master'] = _get_device_index(attributes['master'], ipr)
-            if attributes.has_key('netns'): # ip link set ${dev} netns ${namespace}
+            if attributes.has_key('netns'):  # ip link set ${dev} netns ${namespace}
```

Pylint (3.3.7)

[error] Instance of 'dict' has no 'has_key' member

(E1101)